lib/resources/r4/document_reference.yaml

```diff
@@ -2,10 +2,256 @@
 name: DocumentReference
 field_name_base_url: https://www.hl7.org/fhir/documentreference-definitions.html#DocumentReference
 fields:
+- name: resourceType
+  required: 'Yes'
+  type: string
+  description: The type of the FHIR resource.
+  example: |
+    {
+      "resourceType": "DocumentReference"
+    }
+  note: resourceType must be DocumentReference.
+  url: http://hl7.org/fhir/r4/documentreference.html
+
+- name: authenticator
+  required: 'No'
+  type: Reference
+  action:
+  - create
+  - update
+  description: Who/what authenticated the document
+  note: The authenticator field is only supported with system access
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.authenticator
+  example: |
+    {
+      "authenticator": {
+        "reference": "Practitioner/123456"
+      }
+    }
+
+- name: author
+  required: 'No'
+  type: Reference
+  action:
+    - create
+    - update
+  description: Who and/or what authored the document
+  url: http://hl7.org/fhir/r4/documentreference-definitions.html#DocumentReference.author
+  example: |
+    {
+      "author": [
+        {
+          "reference": "Practitioner/123456"
+        }
+      ]
+    }
+
+- name: subject
+  required: 'Yes'
+  type: Reference
+  action:
+    - create
+  description: Who/what is the subject of the document
```

Comment:
It seems like most (or at least some) of the time we use the same description as the hl7 docs. In this case the hl7 description of the field is more descriptive- we might want to use the hl7 description in most places?

Reply:
With my experience from service request, I was told to keep it brief and sweet. Typically the first sentence in the definition of the element.